hoomd/Communicator.cc

```diff
@@ -1948,8 +1949,14 @@ void Communicator::exchangeGhosts()
             m_velocity_copybuf.resize(max_copy_ghosts);
 
         if (flags[comm_flag::orientation])
-            {
             m_orientation_copybuf.resize(max_copy_ghosts);
+
+        if (flags[comm_flag::angmom])
+            m_angmom_copybuf.resize(max_copy_ghosts);
+
+        if (flags[comm_flag::inertia])
+            {
+            m_inertia_copybuf.resize(max_copy_ghosts);
             }
```

Comment:
For consistency with the rest of the code:
```suggestion
            m_inertia_copybuf.resize(max_copy_ghosts);
```

Reply:
Doing this actually causes an issue when building with ninja. The next lines of code are inside curly brackets and ninja gets confused whether those lines belong to the if statement or not. You can see in the code I edited gave the if statement for orientation the curly brackets presumably for a similar reason. I'm against making this change to avoid getting warnings when building the package.

This is the warning it gives

```
hoomd-blue/hoomd/Communicator.cc: In member function ‘virtual void hoomd::Communicator::exchangeGhosts()’:
hoomd-blue/hoomd/Communicator.cc:1957:9: warning: this ‘if’ clause does not guard... [-Wmisleading-indentation]
 1957 |         if (flags[comm_flag::inertia])
      |         ^~
hoomd-blue/hoomd/Communicator.cc:1960:13: note: ...this statement, but the latter is misleadingly indented as if it were guarded by the ‘if’
 1960 |             {
      |             ^
```

Reply:
Oh I see. Yes, I prefer to always wrap all code under an `if` with the curly brackets, even if they're one-liners, to avoid this ambiguity. If the compiler warns against this, then please follow its instructions.